doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -110,41 +113,53 @@ The presence of a ``bicr`` node in the application devicetree will automatically
 Flashing the BICR binary
 ========================
 
+##TODO
+
 After the |NCS| build system generates the BICR binary, you must flash this binary manually.
 The content of BICR should be loaded to the SoC only once and should not be erased nor modified unless the PCB layout changes.
-To manually flash the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
+To manually program the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
+
+    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex --core Application --serial-number <serial_number>
+
+You only need to follow this programming process once, assuming the PCB configuration applied through the BICR is correct the first time.
+However, it is also possible to reprogram the BICR while in the LCS ``RoT``.
+This can be useful, for example, when adjusting the configuration as the PCB design gets refined or modified, requiring the process to be repeated.
+
+Validate the BICR binary
+------------------------
+
+After programming the BICR binary onto the device, you should validate whether the BICR was successfully installed and executed.
 
-    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Secure --serial-number <serial_number>
+Reset the device
+   The simplest way to check if the BICR is functioning is to reset the device.
+   If the device is still operational after the reset, the BICR is at least partially loaded.
 
-You need to follow this flashing process only one time, as the PCB configuration will not change.
+Validate using nrfutil
+  When in LCS ``EMPTY``, you can use `nrfutil` to explicitly verify the BICR status by reading the memory::
```

Comment:
I don't know this, so i am fumbling a bit in the dark, but:
Must the device be reset before x-read can be done, or can it be done right after loading the BICR, no reset needed in between?
Im asking cause one of the errors says "no found during boot", and that seems to me it would require a reboot before running the check?

Reply:
Actually the address is populated at every step. So there will be a value there if there is not reset. They need to reset. But I  do not know if this is the best way to check.